.github/workflows/flow-temp.yml

```diff
@@ -9,7 +9,7 @@ name: temporary testing
 
 on:
   push:
-    branches-ignore: ['**'] # ignore all branches. Comment this line to run your workflow below on every push.
+    # branches-ignore: ['**'] # ignore all branches. Comment this line to run your workflow below on every push.
```

Comment:
Comment in

Reply:
OK. Changes in flow-temp.yml were reverted.